README.md

```diff
@@ -237,6 +241,18 @@ Both files must be present in the source repository so you should check them int
 
 See [docs/bundler.md](docs/bundler.md) for more details.
 
+### generic fetcher
+
+Generic fetcher is a way for Cachi2 to support pre-fetching arbitrary files that don't fit into other package managers.
```

Comment:
Nitpick: I think we decided to call it "prefetch" (without the hyphen)

Reply:
Fixed.